### PR TITLE
Enhance metadata

### PR DESCRIPTION
**Pull request by kevin-j-smith**

### CODE REVIEW

**Comment by aaronlippold** [comment]
Looks like cof complexity broke

**Comment by aaronlippold** [comment]
Do we have a unit or functional tests for this? It would make me feel a whole lot better :)

File: lib/inspec_tools/inspec.rb [hunk 1]
```diff
@@ -17,14 +17,17 @@
 
 module InspecTools
   class Inspec
-    def initialize(inspec_json, metadata = '{}')
-      @json = JSON.parse(inspec_json.gsub(/\\+u0000/, ''))
```
Comment (lukemalinowski) [inline lib/inspec_tools/inspec.rb]: Do we no longer have issues with nulls in the inspec JSON's?

**Comment by aaronlippold** [comment]
I think in the limit we would like to add the capability to use https://ttytoolkit.org/components/ to implement any user interaction, input for extra data etc as an interactive session - such as metadata - or a CLI flag where we can 'pass in' what the tty-prompt would have them build.

**Comment by Bialogs** [comment]
https://github.com/mitre/inspec_tools/pull/119

**Pull request closed without merging**